### PR TITLE
Fix #2808: MegaMenu start and end template attributes

### LINKED ISSUE #2808: MegaMenu: Start and End templates like MenuBar
### Describe the feature you would like to see added

Logo on MegaMenu similar to Menubar
Sometimes for creating menu with a lot of content MegaMenu is a much better solution
However for styling the menu its not possible to add the app logo at the start of the menu.

### PR DESCRIPTION
**Pull request by melloware**
###Feature Requests
Fix #2808: MegaMenu start and end template attributes

This now brings it in line API wise with the `MenuBar`.

image

### CODE REVIEW

File: components/lib/menubar/Menubar.js [hunk 1]
```diff
@@ -52,18 +52,6 @@ export const Menubar = React.memo(React.forwardRef((props, ref) => {
         useCustomContent
     }));
 
-    const createCustomContent = () => {
```
Comment (melloware) [inline components/lib/menubar/Menubar.js:55]: This method was dead unused code in MenuBar.

**Comment by mertsincan** [comment]
Hi @melloware,

Thanks a lot for this feature! Were you able to test it with orientation="vertical"? Also, can you create a separate demo for this instead of replacing the existing demo?
```
<h5>Templating</h5>
...
```

**Comment by melloware** [comment]
OK good points and I think I definitely need to do more work on vertical may need your help with some CSS 😄

**Comment by melloware** [comment]
Fixed and updated examples.

image

**Pull request merged**